### ISSUE TITLE
Section Option not Generating Routes?

### ISSUE THREAD

**Issue by frodosghost** [prestaconcept/PrestaSitemapBundle#230]
Having a problem with the `section` variable when setting up the sitemap bundle using annotations. Wondering where I am going wrong, any advice?

Route setup is below:

```php
/**
 * @Route("/", name="app_registration_register",
 *     options={"sitemap" = {"priority" = 1, "changefreq" = "monthly", "section"="account"}}
 * )
 */
```

In the base `sitemapindex` the link to the section displays.

```xml
<sitemap>
    <loc>http://localhost:8080/sitemap.account.xml</loc>
    <lastmod>2020-02-28T23:14:19+00:00</lastmod>
</sitemap>
```

Visiting the URL generated returns a 404.

```xml
<response>
    <type>https://tools.ietf.org/html/rfc2616#section-10</type>
    <title>An error occurred</title>
    <status>404</status>
    <detail>Not found</detail>
    <class>
        Symfony\Component\HttpKernel\Exception\NotFoundHttpException
    </class>
    <trace>
        <namespace/>
        <short_class/>
        <class/>
        <type/>
        <function/>
        <file>
            /vagrant/vendor/presta/sitemap-bundle/Controller/SitemapController.php
        </file>
        <line>79</line>
        <args/>
    </trace>
    <trace>
        <namespace>Presta\SitemapBundle\Controller</namespace>
        <short_class>SitemapController</short_class>
        <class>Presta\SitemapBundle\Controller\SitemapController</class>
        <type>
            <![CDATA[ -> ]]>
        </type>
        <function>sectionAction</function>
        <file>/vagrant/vendor/symfony/http-kernel/HttpKernel.php</file>
        <line>145</line>
        <args/>
    </trace>
    <trace>
        <namespace>Symfony\Component\HttpKernel</namespace>
        <short_class>HttpKernel</short_class>
        <class>Symfony\Component\HttpKernel\HttpKernel</class>
        <type>
            <![CDATA[ -> ]]>
        </type>
        <function>handleRaw</function>
        <file>/vagrant/vendor/symfony/http-kernel/HttpKernel.php</file>
        <line>67</line>
        <args/>
    </trace>
    <trace>
        <namespace>Symfony\Component\HttpKernel</namespace>
        <short_class>HttpKernel</short_class>
        <class>Symfony\Component\HttpKernel\HttpKernel</class>
        <type>
            <![CDATA[ -> ]]>
        </type>
        <function>handle</function>
        <file>/vagrant/vendor/symfony/http-kernel/Kernel.php</file>
        <line>191</line>
        <args/>
    </trace>
    <trace>
        <namespace>Symfony\Component\HttpKernel</namespace>
        <short_class>Kernel</short_class>
        <class>Symfony\Component\HttpKernel\Kernel</class>
        <type>
            <![CDATA[ -> ]]>
        </type>
        <function>handle</function>
        <file>/vagrant/public/index.php</file>
        <line>37</line>
        <args/>
    </trace>
</response>
```

**Comment by yann-eugone** [comment]
Hello !
I believe that `account` is not the default section.
Can you try to add an event listener like this one to your project : 

```yml
    account.sitemap_listener:
        class: Presta\SitemapBundle\EventListener\RouteAnnotationEventListener
        arguments : 
            - '@router'
            - 'account'
        tags: ['kernel.event_listener']
```
Does it help ?

**Comment by frodosghost** [comment]
> I believe that `account` is not the default section.

@yann-eugone :: Correct. `default` is still the default section. Was looking at adding another section, so that we could have multiple sections for specific routes.

Is there a way to use multiple sections, defined by annotations?

**Comment by yann-eugone** [comment]
I thought it was possible, but recently, I figured out that it is not possible... The only way to do it is to register these static routes by yourself in a custom event listener.

It is clearly a bug, and I'm not sure to understand why this is not possible.
I was planning to work on a fix on this for the next release.

**Comment by yann-eugone** [comment]
Can you try #231 : 

```
composer require presta/sitemap-bundle:dev-multiple-static-sections@dev
```

**Comment by frodosghost** [comment]
@yann-eugone :: It does work. Does exactly what is needed.

- I was able to keep the current route setup with no `section` configuration, and it shows the "default" sitemap, with listed routes.
- I was able to add a new section, `account`, which created a new sitemap file with specified routes listed within.

**Comment by yann-eugone** [comment]
Released in [`v2.1.0`](https://github.com/prestaconcept/PrestaSitemapBundle/releases/tag/v2.1.0)
